pykafka/producer.py

```diff
@@ -36,26 +44,16 @@
 log = logging.getLogger(__name__)
 
 
-class AsyncProducer():
-    def __init__(self,
-                 topic,
-                 partitioner=None,
-                 compression=CompressionType.NONE,
-                 max_retries=3,
-                 retry_backoff_ms=100,
-                 required_acks=0,
-                 ack_timeout_ms=10000,
-                 batch_size=200,
-                 batch_time_ms=5000,
-                 max_pending_messages=10000):
-        """Create an AsyncProducer for a topic."""
-        raise NotImplementedError("AsyncProducer is unimplemented")
-
-
-class Producer():
+class Producer(object):
     """
-    This class implements the synchronous producer logic found in the
-    JVM driver.
+    This class implements asynchronous producer logic similar to that found in
+    the JVM driver.
```

Comment:
[nitpick] Docstring first lines should be short, one-line, and describe the class. It make code folding and doc generation cleaner. It should also be on the same line as the triple quote.

```
"""Asynchronous Producer Implementation.

Blah blah blah.
"""
```